flow/templates/stampede2.sh

```diff
@@ -20,6 +31,7 @@
 {% endblock %}
 
 {% block body %}
+{% if ns.use_mpi %}
```

Comment:
Can we replace this with `{% ns.use_mpi or operations.length < 2 %}` so that we only use launcher when we have actually more than one operation?

Reply:
I think if we only have one operation, launcher would still be compatible but using mpi would possibly break if the job does not support mpi. On another note, on stampede2 if use_mpi is false and there is only one operation, this is not a good use of the resources and signac will stop the submission unless --force is added right?

Reply:
> I think if we only have one operation, launcher would still be compatible but using mpi would possibly break if the job does not support mpi.

Sorry, I meant `{% ns.use_mpi and operations.length < 2 %}`. I just don't want to use launcher for a single operation.

> On another note, on stampede2 if use_mpi is false and there is only one operation, this is not a good use of the resources and signac will stop the submission unless --force is added right?

You could have a single operation that is "manually" parallelized without MPI.

Reply:
Hmm that logic doesn't seem quite right. That would cause us to go the launcher route if we were submitting multiple MPI operations, right? I think your original suggestion with `or` is correct. Pengji raises a valid point, but I don't think it's a concern because `mpi_prefix` is only getting set within this block if `operations.directives.nranks` is set, so `ibrun` won't be inserted if the job is not requesting multiple ranks.

I think we should rename the variable from `use_mpi` to `use_launcher` or so, because it's actually a catch-all for all the possible parallel behaviors that launcher won't support.